service/src/main/java/com/epam/esm/service/utils/CertificateParameterApiValidator.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.esm.service.utils;
+
+import com.epam.esm.model.parameters.CertificateParameter;
+
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CertificateParameterApiValidator {
+    private static final String NAME_PATTERN = "[a-zA-Zа-яА-ЯёЁ0-9\\s?!,.:'\\-]{3,45}";
+    private static final String DESCRIPTION_PATTERN =  "[a-zA-Zа-яА-ЯёЁ0-9\\s?!,.:'\\-]{1,1000}";
+    private static final String PARAMETER_SORT = "sort";
+
+    public static boolean isValidParams(Map<String, String> groupParameters) {
+        Map<String, String> groupParams = groupParameters.entrySet().stream().
```

Comment:
code duplication for getting this `groupParams`

Reply:
Will be fixed in the next PR